frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -221,7 +221,7 @@ frame_benchmarking::benchmarks! {
 		let initial_balance = T::Currency::minimum_balance() * 10u32.into();
 		T::Currency::make_free_balance_be(&receiver, initial_balance);
 		let ready = ReadySolution {
-			supports: vec![],
+			supports: vec![].try_into().unwrap(),
```

Comment:
```suggestion
			supports: Default::default()
```

and then you can do `let ready = ReadySolution::default()`.